plugins/powerd/fu-plugin-powerd.c

```diff
@@ -69,6 +96,11 @@ fu_plugin_startup(FuPlugin *plugin, GError **error)
 {
 	FuPluginData *data = fu_plugin_get_data(plugin);
 	g_autofree gchar *name_owner = NULL;
+	gboolean file_absent = fu_plugin_powerd_create_suspend_file(error);
+	if (!file_absent) {
+		g_prefix_error(error, "lock file unable to be deleted");
+		return FALSE;
+	}
```

Comment:
*create* on startup? I can' kinda see the need for `fu_plugin_powerd_delete_suspend_file` but I can't see the need for the opposite. I'd just drop these 5 lines completely; if fwupd crashes half way thought updating you have bigger problems then a stale lock file..

Reply:
Agh that was supposed to be delete yeah. Is there a problem with having it there? Despite having bigger problems I feel like that is still a problem. Also, this is also useful for testing, when updates may purposefully stopped in the middle.